### ISSUE TITLE
Dialog log not running

### ISSUE THREAD

**Issue by JeromSar** [derpfacedirk/Goldiriath#74]
From https://github.com/Goldiriath/Configuration/issues/12
derpfacedirk wrote:
```
The dialog I wrote for the hopeless painter isn't running
no errors.
I have no idea why
@JeromSar any idea's?
```

**Comment by JeromSar** [comment]
@derpfacedirk Not sure. Are you sure you have the following set up correctly?
* NPC ID is correct (as linked with the NPC ID citizens provides you).
* Dialog script is linked to the 'click' option in the dialog file.

**Comment by derpfacedirk** [comment]
yeah, Id and name are the same. and click is linked to correct places
other dialogs are running so i probably did something wrong but I can't think of what
